### ISSUE TITLE
"wicked" network service starting

### ISSUE THREAD

**Issue by kkaempf** [rancher/elemental#1138]
There's still `wicked` being started at bootup. This needs to go, only NetworkManager is supported.

**Comment by kkaempf** [comment]
`wicked` and `wicked-service` are installed, pulled in by some dependency 🙈 

Will fix it together with #1141 🤞🏻

**Comment by kkaempf** [comment]
It's a new dependency of `dracut` 🤦🏻 
~~~
> rpm -q --requires dracut | grep wic
(iputils or arping2 or wicked)
~~~

so adding `iputils` should fix that ...
